.editorconfig

```diff
@@ -0,0 +1,11 @@
+; EditorConfig file: http://EditorConfig.org
+; Install the "EditorConfig" plugin into your editor to use
+
+root = true
+
+[*]
+charset = utf-8
+insert_final_newline = true
+indent_style = space
+indent_size = 2
+trim_trailing_whitespace = true
```

Comment:
A [Rider](https://en.wikipedia.org/wiki/Rider_(legislation)) :)

Looks good to me though.